heartbeat/_meta/beat.yml

```diff
@@ -182,6 +182,23 @@ heartbeat.monitors:
     # Required response contents.
     #body:
 
+# Configure monitors default settings shared by all monitors.
+heartbeat.defaults.monitors:
```

Comment:
doesn't really matter if commented out or not. In either case it will be `nil` config.